test/unittests/mongoBackend/mongoContextProvidersQuery_test.cpp

```diff
@@ -563,11 +564,37 @@ TEST(mongoContextProvidersQueryRequest, noPatternAttrsAll)
   /* Check response is as expected */
   EXPECT_EQ(SccOk, ms);
 
-  EXPECT_EQ(SccFound, res.errorCode.code);
-  EXPECT_EQ("Found", res.errorCode.reasonPhrase);
-  EXPECT_EQ("http://cr1.com", res.errorCode.details);
+  EXPECT_EQ(SccNone, res.errorCode.code);
```

Comment:
If the value is SccOk by default and SccOk is part of the valid values that mongoBackend may set, you cannot distinguish between this situations: 1) the result was actually SccOk, 2) the result was not SccOk but mongoBackend didn't change the value due to some internal error.

However, if you use as default a value SccNone you can detect case 2)

In the case of re-visiting this, I'd do that process outside the scope of this PR (otherwise, the PR could be delayed while discussion progress).

NTC